pkg/sbproxy/notifications/handlers/visibilities_handler.go

```diff
@@ -152,116 +146,126 @@ func (vnh *VisibilityResourceNotificationsHandler) OnUpdate(ctx context.Context,
 	if (len(labelsToRemove) == 0 && newVisibilityPayload.Resource.PlatformID == "") || (len(labelsToRemove) != 0 && newVisibilityPayload.Resource.PlatformID != "") {
-		log.C(ctx).Infof("Attempting to disable access for plan with catalog ID %s for platform broker with name %s and labels %v...", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName, labelsToRemove)
-
-		if err := vnh.VisibilityClient.DisableAccessForPlan(ctx, &platform.ModifyPlanAccessRequest{
-			BrokerName:    platformBrokerName,
-			CatalogPlanID: newVisibilityPayload.Additional.ServicePlan.CatalogID,
-			Labels:        labelsToRemove,
-		}); err != nil {
-			log.C(ctx).WithError(err).Errorf("error disabling access for plan %s in broker with name %s", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName)
-			return err
-		}
-		log.C(ctx).Infof("Successfully disabled access for plan with catalog ID %s for platform broker with name %s and labels %v...", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName, labelsToRemove)
+		return vnh.disableAccessForPlan(ctx, platformBrokerName, newVisibilityPayload.Additional.ServicePlan.CatalogID, labelsToRemove)
 	}
 	return nil
 }
 
 func (vnh *VisibilityResourceNotificationsHandler) enableServiceAccess(ctx context.Context, labelsToAdd types.Labels, newVisibilityPayload visibilityWithAdditionalDetails, platformBrokerName string) error {
 	if (len(labelsToAdd) == 0 && newVisibilityPayload.Resource.PlatformID == "") || (len(labelsToAdd) != 0 && newVisibilityPayload.Resource.PlatformID != "") {
-		log.C(ctx).Infof("Attempting to enable access for plan with catalog ID %s for platform broker with name %s and labels %v...", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName, labelsToAdd)
-
-		if err := vnh.VisibilityClient.EnableAccessForPlan(ctx, &platform.ModifyPlanAccessRequest{
-			BrokerName:    platformBrokerName,
-			CatalogPlanID: newVisibilityPayload.Additional.ServicePlan.CatalogID,
-			Labels:        labelsToAdd,
-		}); err != nil {
-			log.C(ctx).WithError(err).Errorf("error enabling access for plan %s in broker with name %s", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName)
-			return err
-		}
-		log.C(ctx).Infof("Successfully enabled access for plan with catalog ID %s for platform broker with name %s and labels %v...", newVisibilityPayload.Additional.ServicePlan.CatalogID, platformBrokerName, labelsToAdd)
+		return vnh.enableAccessForPlan(ctx, platformBrokerName, newVisibilityPayload.Additional.ServicePlan.CatalogID, labelsToAdd)
 	}
 	return nil
 }
 
+func (vnh *VisibilityResourceNotificationsHandler) enableAccessForPlan(
+	ctx context.Context, platformBrokerName, catalogPlanID string, labels types.Labels,
+) error {
+	logger := log.C(ctx)
+	logger.Infof("Attempting to enable access for plan with catalog ID %s for platform broker with name %s ...",
```

Comment:
Can we also log the catalog name of the plan? It would be more user friendly (even though the ID is sufficient).
Something like `Attempting to enable access for plan %s with catalog ID %s`